.github/workflows/build.yaml

```
@@ -43,23 +44,22 @@ jobs:
           registry: ${{ env.REGISTRY }}
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      -
-        name: Build and push
+
+      - name: Build and push
         uses: docker/build-push-action@v4
         with:
-          context: "{{defaultContext}}:php-nginx/${{ matrix.php }}"
-          tags: "${{ env.IMAGE_NAME }}:${{ matrix.php}}"
+          context: "{{defaultContext}}:${{ matrix.type }}/${{ matrix.version }}"
```

actionlint (1.7.7)

property "type" is not defined in object type {}

(expression)